src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java

```diff
@@ -928,6 +929,62 @@ public HyperwalletWebhookNotification getWebhookEvent(String webhookToken) {
     }
 
     //--------------------------------------
+    // Transfer Methods
+    //--------------------------------------
+
+    /**
+     * Create a Transfer Method
+     *
+     * @param jsonCacheToken String JSON cache token
+     * @param transferMethod TransferMethod object to create
+     * @return HyperwalletTransferMethod Transfer Method object created
+     */
+    public HyperwalletTransferMethod createTransferMethod(String jsonCacheToken, HyperwalletTransferMethod transferMethod) {
+
+        if (transferMethod == null || StringUtils.isEmpty(transferMethod.getUserToken())) {
+            throw new HyperwalletException("User token is required");
+        }
+        if (StringUtils.isEmpty(jsonCacheToken)) {
+            throw new HyperwalletException("JSON token is required");
+        }
+        transferMethod = copy(transferMethod);
+
+        transferMethod.setToken(null);
+        transferMethod.setStatus(null);
+        transferMethod.setCreatedOn(null);
+
+        HashMap<String, String> headers = new HashMap<String, String>();
+        headers.put("Json-Cache-Token", jsonCacheToken);
+
+        return apiClient.post(url + "/users/"+transferMethod.getUserToken()+"/transfer-methods", transferMethod, HyperwalletTransferMethod.class, headers);
+    }
+
+
+    /**
+     * Create a Transfer Method
+     *
+     * @param jsonCacheToken String JSON cache token
+     * @param userToken String user token
+     * @return HyperwalletTransferMethod Transfer Method object created
+     */
+    public HyperwalletTransferMethod createTransferMethod(String jsonCacheToken, String userToken) {
+
+        if (StringUtils.isEmpty(userToken)) {
+            throw new HyperwalletException("User token is required");
+        }
+        if (StringUtils.isEmpty(jsonCacheToken)) {
+            throw new HyperwalletException("JSON token is required");
+        }
+
+        HyperwalletTransferMethod transferMethod = new HyperwalletTransferMethod();
+        transferMethod.setUserToken(userToken);
```

Comment:
Yep this method for me should be removed since even if we set a default but in this context we cannot know for sure what are the users available transfer methods or the concept of default transfer method to a specific user given only the `user token`, please also confirm with @fkrauthan-hyperwallet thanks.